interfaces/cython/cantera/examples/onedim/diffusion_flame.py

```diff
@@ -22,7 +23,7 @@
 
 # Create the gas object used to evaluate all thermodynamic, kinetic, and
 # transport properties.
-gas = ct.Solution('gri30.xml', 'gri30_mix')
+gas = ct.Solution("gri30.yaml", "gas", transport_model="Mix")
```

Comment:
Shouldn't this just be 
```suggestion
gas = ct.Solution("gri30.yaml")
```

Reply:
I removed the phase specification here. I addressed the `transport_model` argument in another comment.